envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -205,31 +212,12 @@ async def dep_date_check(
         elif await dep.release_date_mismatch:
             self.error(
                 "release_dates",
```

Comment:
self.active_check ?

Reply:
yep, makes sense - ive been meaning to abstract this so its not necessary at all - but its a bit of a breaking change

ill update the checks in this file, and related tests